pkg/config/config.go

```diff
@@ -439,23 +439,31 @@ type SnapcraftAppMetadata struct {
 	RestartCondition string `yaml:"restart_condition,omitempty"`
 }
 
+type SnapcraftLayoutMetadata struct {
+	Symlink  string `yaml:",omitempty"`
+	Bind     string `yaml:",omitempty"`
+	BindFile string `yaml:"bind-file,omitempty"`
```

Comment:
```suggestion
	BindFile string `yaml:"bindFile,omitempty"`
```

on goreleaser we always use snakeCase for config

Reply:
ok, change `bind-file` to `bind_file`